common/handy/pointers.go

```diff
@@ -0,0 +1,21 @@
+package handy
+
+var Pointers = pointers{} // nolint:gochecknoglobals
+
+type pointers struct{}
+
+func (pointers) Str(input string) *string {
```

Comment:
Please add comment strings to each of these methods in this file so other people using these functions can get IDE hints.